game-core/src/test/java/games/strategy/engine/data/gameparser/XmlGameElementMapperTest.java

```diff
@@ -22,59 +19,47 @@
 /**
  * Simple test of XmlGameElementMapper to verify error handling and a quick check of the happy case.
  */
-public class XmlGameElementMapperTest {
+class XmlGameElementMapperTest {
   private static final String NAME_THAT_DOES_NOT_EXIST = "this is surely not a valid identifier";
 
   private XmlGameElementMapper testObj;
 
-  @BeforeAll
-  public static void disableErrorPopup() {
-    ClientLogger.disableErrorPopupForTesting();
-  }
-
-  @AfterAll
-  public static void resetErrorPopup() {
-    // we do this to make sure any other tests will show the popup and have a similar method
-    // call. We do not want test ordering to be important.
-    ClientLogger.resetErrorPopupForTesting();
-  }
-
   @BeforeEach
-  public void setup() {
+   void setup() {
```

Comment:
Might want to run the formatter in this file.  Looks like there's an extra leading space before all the `void` tokens here and below.

Reply:
done; 6da1de4